arrays_in_javaScript/HW7_task2.js

```diff
@@ -0,0 +1,4 @@
+const numbers = [9, 3, 7, 4, 0, 11];
+
+const newNumbers = numbers.map((number) => number * numbers.indexOf(number))
```

Comment:
```suggestion
const newNumbers = numbers.map((number, index) => number * index)
```
Не по змісту завдання, але так більш органічно буде